src/components/Home/Home.jsx

```diff
@@ -1,27 +1,73 @@
 import React from 'react';
-import { Redirect, useHistory } from 'react-router-dom';
+import { useState } from 'react';
+import { useForm } from '../../hooks/useForm';
+import { useFirebase } from '../../hooks/useFirebase.js';
+import { useHistory } from 'react-router-dom';
 import getToken from '../../lib/tokens';
 
 const Home = () => {
-  const userToken = localStorage.getItem('token');
+  const [values, handleInputChange, setValues] = useForm({
+    token: '',
+  });
+
+  const [msn, setMsn] = useState('');
```

Comment:
what is `msn`? I typically try to avoid all abbreviations or shortening of words b/c it can cause some confusion.

Reply:
Thanks, it has been solved.👍🏻